projects/demo/src/modules/icons/marker-icon/examples/2/index.less

```diff
@@ -1,4 +1,4 @@
-@import 'taiga-ui-local';
+@import '../../../../../../../core/styles/taiga-ui-local';
```

Comment:
?

Reply:
If the initial import doesn't work... Should we fix something inside `project.json`?

Reply:
fixed